### ISSUE TITLE
[Bug]: video page "reload" window button is missing

### ISSUE THREAD

**Issue by dan-software-engineer** [FreeTubeApp/FreeTube#8946]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.
- [x] I have read and agree to follow the [rules](https://docs.freetubeapp.io/community/rules/).

### Describe the bug

Steps:

1. Open a video page.
2. The FreeTube app shows an error on top of the video player, for example some network error, etc (the type of error is irrelevant for the purpose of this ticket).
3. Fix the external issue whatever it is (for example restart the Wi-Fi, etc - the type of fix is irrelevant for the purpose of this ticket).
4. the main issue: the FreeTube app is still stuck on the error message and there is absolutely no way to reload the page to make the error go away. Someone suggested in a comment in a related ticket https://github.com/FreeTubeApp/FreeTube/issues/8919 to navigate back and then to navigate forward to reload the video page. This is a very bad solution from an user experience point of view - imagine having no "reload" button in Google Chrome or in Mozilla Firefox and having to navigate back and forth every time you want to reload a page. This makes no sense from an user experience point of view.

Please add add a "reload" button, for example next to the back and forth navigation buttons or next to the "open new window" button in the video page header.

Thanks!

### Expected Behavior

Please add add a "reload" button, for example next to the back and forth navigation buttons or next to the "open new window" button in the video page header.

Thanks!

### Issue Labels

accessibility issue

### FreeTube Version

v0.24 beta

### Operating System Version

windows 11 25h2

### Installation Method

.exe

### Primary API used

Local API

### Additional Information

**To moderators and contributors - please don't block people's ability to comment on issues if you personally disagree with an issue, because if you do that, people will open duplicate tickets just to be able to comment on the original issue. I suggest letting people (politely) voice their concerns in comments section even if you feel that the issue is not relevant for you personally. Thank you very much.**

### Nightly Build

- [x] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by dan-software-engineer** [comment]
To moderators and contributors - please don't block people's ability to comment on issues if you personally disagree with an issue, because if you do that, people will open duplicate tickets just to be able to comment on the original issue. I suggest letting people (politely) voice their concerns in comments section even if you feel that the issue is not relevant for you personally. Thank you very much.
